ui/src/app/components/sharded-dataflow-migration-details-form/sharded-dataflow-migration-details-form.component.ts

```diff
@@ -394,7 +406,7 @@ export class ShardedDataflowMigrationDetailsFormComponent implements OnInit {
 
   determineFormValidity(): boolean {
     if (this.migrationProfileForm.valid && (this.selectedSourceProfileOption !== Profile.NewConnProfile || this.createSrcConnSuccess) &&
-    (this.selectedTargetProfileOption !== Profile.NewConnProfile || this.createTgtConnSuccess)) {
+    (this.selectedTargetProfileOption !== Profile.NewConnProfile || this.createTgtConnSuccess) && (this.migrationProfileForm.value.inputType === "text" === this.verifyJson)) {
```

Comment:
Didn't understand the If clause added , is it equivalent to `(this.migrationProfileForm.value.inputType === "text") && this.verifyJson`

Reply:
Yes